cmd/agent/common/loader.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/DataDog/datadog-agent/comp/core/autodiscovery"
 	"github.com/DataDog/datadog-agent/comp/core/config"
+
```

Comment:
Half the files changed are just adding an empty line 😄

Reply:
Yeah, something went wrong with cherry-pick some commits. I can fix it if we like the approach